lib/td_df_lib/format.ex

```diff
@@ -194,11 +194,9 @@ defmodule TdDfLib.Format do
   end
 
   defp set_search_value(%{"name" => name, "type" => "url"}, acc) do
```

Comment:
Consider if the rest of set_search_value/2 functions require similar adjustments.

Reply:
Also define if default values should have system as origin